packages/ember-runtime/lib/computed/reduce_computed_macros.js

```diff
@@ -13,7 +13,7 @@ import { isArray } from 'ember-runtime/utils';
 
 function reduceMacro(dependentKey, callback, initialValue) {
   return computed(`${dependentKey}.[]`, function() {
-    return get(this, dependentKey).reduce(callback, initialValue);
+    return get(this, dependentKey).reduce(callback.bind(this), initialValue);
```

Comment:
:trollface: let the phantomjs trolling begin (`Function.prototype.bind` is not available).